docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Inline Automatic Completions
+lang: en
+---
+
+## Overview
+
+Use AI-based autocomplete to accept suggestions for repetitive, boilerplate code right in your editor.
+
+## Set Up Inline Autocomplete
+
+To use inline autocomplete, first activate the feature:
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
```

Comment:
The feature will be default "On" for users so they shouldn't _need_ to do this, but we can point it out as it's always disable-able